scripts/scripts.js

```diff
@@ -70,6 +82,15 @@ async function loadEager(doc) {
     document.body.classList.add('appear');
     await waitForLCP(LCP_BLOCKS);
   }
+
+  try {
+    /* if desktop (proxy for fast connection) or fonts already loaded, load fonts.css */
```

Comment:
> fast connection

It's not widely supported yet but you could use `navigator.connection.downlink` to get the bandwidth estimate in megabits per second to know if the user has a fast connection or not https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/downlink